console/console-init/ui/src/modules/project-detail/components/GeneralInfo/GeneralInfo.tsx

```diff
@@ -22,50 +24,98 @@ const styles = StyleSheet.create({
     fontSize: 20
   }
 });
-export interface IInfoTypePlan {
-  type: string;
-  plan?: string;
-}
-
 interface IGeneralInfoProps {
   addressSpace: string;
-  eventAddressName: IInfoTypePlan;
-  telemetryAddressName: IInfoTypePlan;
-  commandAddressName: IInfoTypePlan;
+  eventAddresses: Array<string>;
+  telemetryAddresses: Array<string>;
+  commandAddresses: Array<string>;
   maxConnection: number;
   dataVolume: number;
   startDate: string | Date;
   endDate: Date | string;
+  namespace: string;
 }
 
 const GeneralInfo: React.FunctionComponent<IGeneralInfoProps> = ({
   addressSpace,
-  eventAddressName,
-  telemetryAddressName,
-  commandAddressName,
+  eventAddresses,
+  telemetryAddresses,
+  commandAddresses,
   maxConnection,
   dataVolume,
   startDate,
-  endDate
+  endDate,
+  namespace
 }) => {
   return (
     <PageSection>
       <Card>
-        <CardHeader className={css(styles.font_size)}>General Info</CardHeader>
+        <CardHeader className={css(styles.font_size)}>
+          <Title size="xl" headingLevel="h2">
+            General Info
+          </Title>
+        </CardHeader>
         <CardBody>
           <b className={css(styles.style_margin)}>Address space</b>
-          {addressSpace} <br />
-          <b className={css(styles.style_margin)}>Event address name</b>
-          {eventAddressName.type}{" "}
-          {eventAddressName.plan ? ", " + eventAddressName.plan : ""}
+          <NavLink
+            //TODO:=modify route
+            to={`/address-spaces/${namespace}/${addressSpace}/standard/addresses`}
+            className="pf-c-nav__link"
+            id={`navlink-as-${addressSpace}`}
+          >
+            {addressSpace}
+          </NavLink>{" "}
+          <br />
+          <b className={css(styles.style_margin)}>Events address name</b>
+          {eventAddresses &&
+            eventAddresses.length > 0 &&
+            eventAddresses.map((address: string) => (
+              <>
+                <NavLink
+                  //TODO:=modify route
+                  to={""}
+                  className="pf-c-nav__link"
+                  id={`navlink-gi-event-${address}`}
+                  key={`navlink-gi-event-${address}`}
+                >
+                  {address}
+                </NavLink>{" "}
+              </>
+            ))}
           <br />
           <b className={css(styles.style_margin)}>Telemetry address name</b>
-          {telemetryAddressName.type}
-          {telemetryAddressName.plan ? ", " + telemetryAddressName.plan : ""}
+          {telemetryAddresses &&
+            telemetryAddresses.length > 0 &&
+            telemetryAddresses.map((address: string) => (
```

Comment:
can we try to create generic component/function in same file  for NavLink  telemetryAddresses and commandAddresses. both have same kind of logic. Component/function name can be like  NavLinkList.

Reply:
Will create in future when we declare the routes.